package.json

```diff
@@ -59,8 +59,8 @@
 		"npm": "^6.12.1"
 	},
 	"scripts": {
-		"preanalyze-bundles": "cross-env-shell NODE_ENV=production CALYPSO_ENV=production EMIT_STATS=true PROGRESS=true npm run -s build-client-evergreen",
-		"analyze-bundles": "webpack-bundle-analyzer client/stats.json public/evergreen -h 127.0.0.1 -p 9898 -s gzip",
+		"analyze-bundles": "npm run build-client-stats && webpack-bundle-analyzer client/stats.json public/evergreen -h 127.0.0.1 -p 9898 -s gzip",
+		"analyze-icfy": "npm run build-client-stats && node --max-old-space-size=4096 bin/icfy-analyze.js && mv client/stats.json client/chart.json \"$CIRCLE_ARTIFACTS/icfy\"",
```

Comment:
If there's an issue with `cross-env-shell` that results in a `0` exist status when we expect non-`0`, this type of chaining will fail to capture that, won't it?

Reply:
Yes, that's true. I discovered the `cross-env` issue only after migrating the `icfy-analyze` task to a NPM script and debugging it.

In other words, I don't expect this PR to solve any of the problems, but the refactoring still seems like a good idea to me 🙂